Doc/library/collections.rst

```diff
@@ -358,7 +358,7 @@ Common patterns for working with :class:`Counter` objects::
     list(c)                         # list unique elements
     set(c)                          # convert to a set
     dict(c)                         # convert to a regular dictionary
-    c.items()                       # convert to a list of (elem, cnt) pairs
+    c.items()                       # convert to a view object of the dictionary’s items (elem, cnt) pairs
```

Comment:
Maybe just

```suggestion
    c.items()                       # convert to a iterator of (elem, cnt) pairs
```

?

Reply:
It's not an iterator.

I'd just change "list" to "view". And "convert to" to "create", since the view doesn't really contain the data, so "convert to" seems wrong to me.

Reply:
Yes, it is just an iterable 👍 

```
>>> {}.items().__iter__
<method-wrapper '__iter__' of dict_items object at 0x102cd1250>
>>> {}.items().__next__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'dict_items' object has no attribute '__next__'. Did you mean: '__ne__'?
```